backend/src/main/java/org/cryptomator/hub/api/UsersResource.java

```diff
@@ -158,24 +163,46 @@ public Response updateMyAccessTokens(@NotNull Map<UUID, String> tokens) {
 	@NoCache
 	@Transactional
 	@Operation(summary = "get the logged-in user")
+	@Parameter(name = "withLastAccess", in = ParameterIn.QUERY, description = "adds last access values to the devices (if present)")
 	@APIResponse(responseCode = "200", description = "returns the current user")
 	@APIResponse(responseCode = "404", description = "no user matching the subject of the JWT passed as Bearer Token")
 	public UserDto getMe(@QueryParam("withDevices") boolean withDevices, @QueryParam("withLastAccess") boolean withLastAccess) {
 		User user = userRepo.findById(jwt.getSubject());
-		Set<DeviceResource.DeviceDto> deviceDtos;
+		Set<DeviceResource.DeviceDto> deviceDtos = new HashSet<>();
```

Comment:
```suggestion
		Set<DeviceResource.DeviceDto> deviceDtos = Set.of();
```

Reply:
Or even better: Leave it uninitialized and add a separate else branch to init to empty (immutable) set. Probably gives us the most resilient compile-time checks.

Reply:
Fixed in 00bdeb1